CMakeLists.txt

```diff
@@ -736,6 +744,11 @@ install(
         "${CMAKE_CURRENT_SOURCE_DIR}/README.md"
   DESTINATION "${CMAKE_INSTALL_DOCDIR}")
 
+
+# -- packaging -----------------------------------------------------------------
+
+include(Package)
```

Comment:
Can you rename the file from `Package.cmake` to `VASTPackage.cmake`? This has become common practice to be able to disambiguate which project a CMake include came from.